ui/admin/tests/e2e/tests/auth-method-ldap.spec.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/* eslint-disable no-undef */
+const { test, expect } = require('@playwright/test');
+const { execSync } = require('child_process');
+const { checkEnv, authenticatedState } = require('../helpers/general');
+const { nanoid } = require('nanoid');
+
+const {
+  authenticateBoundaryCli,
+  checkBoundaryCli,
+  deleteOrgCli,
+} = require('../helpers/boundary-cli');
+const {
+  createNewOrg,
+  createRole,
+  addPrincipalToRole,
+} = require('../helpers/boundary-ui');
+
+test.use({ storageState: authenticatedState });
+
+test.beforeAll(async () => {
+  await checkEnv([
+    'E2E_LDAP_ADDR',
+    'E2E_LDAP_DOMAIN_DN',
+    'E2E_LDAP_ADMIN_DN',
+    'E2E_LDAP_ADMIN_PASSWORD',
+    'E2E_LDAP_USER_NAME',
+    'E2E_LDAP_USER_PASSWORD',
+    'E2E_LDAP_GROUP_NAME',
+  ]);
+
+  await checkBoundaryCli();
+});
+
+test('Set up LDAP auth method @ce @ent @docker', async ({ page }) => {
+  await page.goto('/');
+  let org;
+  let connect;
+  try {
+    // Create an org
+    const orgName = await createNewOrg(page);
+    await authenticateBoundaryCli();
+    const orgs = JSON.parse(execSync('boundary scopes list -format json'));
+    org = orgs.items.filter((obj) => obj.name == orgName)[0];
+
+    // Navigate to LDAP auth method creation page
+    await page
+      .getByRole('navigation', { name: 'IAM' })
+      .getByRole('link', { name: 'Auth Methods' })
+      .click();
+    await page.getByRole('button', { name: 'New' }).click();
+    await page.getByRole('link', { name: 'LDAP' }).click();
+
+    // Fill out the form
+    const ldapAuthMethodName = 'LDAP ' + nanoid();
+    await page.getByLabel('Name').fill(ldapAuthMethodName);
+    await page.getByLabel('Description').fill('LDAP Auth Method');
+    await page.getByLabel('Address').fill(process.env.E2E_LDAP_ADDR);
+    await page.getByLabel('Bind DN').fill(process.env.E2E_LDAP_ADMIN_DN);
+    await page
+      .getByLabel('Bind Password')
+      .fill(process.env.E2E_LDAP_ADMIN_PASSWORD);
+    await page.getByRole('switch', { name: 'Discover DN' }).click();
+    await page.getByLabel('User DN').fill(process.env.E2E_LDAP_DOMAIN_DN);
+    await page.getByLabel('User Attribute').fill('uid');
+    await page.getByLabel('Group DN').fill(process.env.E2E_LDAP_DOMAIN_DN);
```

Comment:
Just to double check, are the User DN and Group DN supposed to be the same?

Reply:
Yup. I believe it describes what part of the "LDAP tree" users and groups live. In our case, I've set up the LDAP server for users and groups to live in the same part of the tree.